go/libkb/loaduser.go

```diff
@@ -222,7 +222,7 @@ func (arg *LoadUserArg) resolveUID() (ResolveResult, error) {
 		return rres, fmt.Errorf("resolveUID: no uid or name")
 	}
 
-	if rres = arg.G().Resolver.ResolveWithBody(arg.name); rres.err != nil {
+	if rres = arg.G().Resolver.ResolveWithBody(arg.name).FailOnDeleted(); rres.err != nil {
```

Comment:
Does this mean the api changes? I think this wouldn't have errored before.

Reply:
Can you ask again? Not sure I understand the question.

Reply:
Oh, sorry - my current understanding of the code was that `func (arg *LoadUserArg) resolveUID()` would not have returned an error if user was deleted before we started working on this issue, but now it will. Did I get that correctly?

Reply:
Yes I think you are correct.

Reply:
OK I think it's fine if the test passed, and I think we will catch bugs quickly.

When switching these things around I had failures in different places but they were apparent by just running go tests.